packages/utils/odsp-doclib-utils/src/test/odspErrorUtils.spec.ts

```diff
@@ -153,7 +153,8 @@ describe("OdspErrorUtils", () => {
 			enrichOdspError(error);
 
 			assert(typeof error.online === "string");
-			assert(isILoggingError(error));
+			// The forced typing above causes this line to result in error having type 'never' unless we cast as unknown
+			assert(LoggingError.isLoggingError(error as unknown));
```

Comment:
One funky case of moving to a class over an interface

Reply:
classes also get a bit weird when you cross compact boundaries, like driver, loader, runtime, or datastore, since different layer can have different copies. interfaces are nice, in that there is never the assumption of a specific implementation